### PR TITLE
Lingraph Api Side changes

### PR DESCRIPTION
**Pull request by StrikerSumit39**
New Line model in Bargraph

### CODE REVIEW

File: CFStats/CFUserInterface/UiModels/ApiHandlerModels.cs [hunk 1]
```diff
@@ -11,4 +11,13 @@ public class ProblemModel
         public int wrongAttempts { get; set; }
         public int correctAttempts { get; set; }
     }
+
+    public class ContestModel
+    {
+        public string ContestId { get; set; }
+        public string ContestName { get; set; }
```
Comment (SuveenKumar) [inline CFStats/CFUserInterface/UiModels/ApiHandlerModels.cs:18]: Make ratingchange and rank as int variable

File: CFStats/CFUserInterface/Common/ApiHandler.cs [hunk 2]
```diff
@@ -90,8 +90,20 @@ private static void FillContestSet()
 
             foreach (var contest in ApiControl.userContests.result)
             {
-                contestMap[contest.ratingUpdateTimeSeconds.ToString()] = contest.newRating.ToString();
-            }                  
+                ContestModel contestLineData = new ContestModel();
+                contestLineData.ContestId = contest.contestId;
```
Comment (SuveenKumar) [inline CFStats/CFUserInterface/Common/ApiHandler.cs:94]: Make contestLineData to contestData

**Review by SuveenKumar — APPROVED** [review]

File: CFStats/CFUserInterface/Common/ApiHandler.cs [hunk 3]
```diff
@@ -90,8 +90,20 @@ private static void FillContestSet()
...
+                contestLineData.ContestName = contest.contestName;
+                contestLineData.ContestRank = contest.rank;
+                contestLineData.RatingChange = (Convert.ToInt32((contest.newRating)) - Convert.ToInt32(contest.oldRating)).ToString();
+
+
+                contestMap[contest.ratingUpdateTimeSeconds.ToString()] = contestLineData;
+
```
Comment (SuveenKumar) [inline CFStats/CFUserInterface/Common/ApiHandler.cs:101]: Remove Console.writle

File: CFStats/CFUserInterface/Common/ApiHandler.cs [hunk 4]
```diff
@@ -90,8 +90,20 @@ private static void FillContestSet()
...
+                Console.WriteLine(contestLineData.ContestId);
```
Comment (SuveenKumar) [inline CFStats/CFUserInterface/Common/ApiHandler.cs:102]: Remove Console.WriteLine statements

**Review by SuveenKumar — APPROVED** [review]

File: CFStats/CFUserInterface/UiModels/ApiHandlerModels.cs (hunk 1 above)
Comment (StrikerSumit39) [reply CFStats/CFUserInterface/UiModels/ApiHandlerModels.cs:19]: done

File: CFStats/CFUserInterface/Common/ApiHandler.cs (hunk 4 above)
Comment (StrikerSumit39) [reply CFStats/CFUserInterface/Common/ApiHandler.cs]: done

File: CFStats/CFUserInterface/Common/ApiHandler.cs (hunk 2 above)
Comment (StrikerSumit39) [reply CFStats/CFUserInterface/Common/ApiHandler.cs]: done

**Comment by StrikerSumit39** [comment]
All Changes are done

**Pull request merged**